### DISCUSSION TITLE
why i paid in gitkraken and still can not use it in vscode

### DISCUSSION THREAD

**Discussion by serene-ding** [gitkraken/vscode-gitlens#2914] [Q&A] [answered]
![image](https://github.com/gitkraken/vscode-gitlens/assets/77049532/039ddcbc-5701-4f50-8bf8-d4768c414854)
![image](https://github.com/gitkraken/vscode-gitlens/assets/77049532/e650fd0c-6276-4f51-9ebb-968033ed3a3d)

**Comment by d13** [comment, marked as answer]
Could you try signing out of your GitKraken account in GitLens and then sign back in?

**Reply by serene-ding** [reply]
yes it worked
![image](https://github.com/gitkraken/vscode-gitlens/assets/77049532/ffa346b0-393e-4af6-8ae1-ea0d03179c11)
by the way i paid twice by mistake. can i get a refund?

**Reply by serene-ding** [reply]
i canceled one of them.

**Reply by eamodio** [reply]
Please contact support as they should be able to help you with that: https://help.gitkraken.com/gitlens/gl-contact-support/
